src/metrics/prometheus_histogram.erl

```diff
@@ -241,25 +241,23 @@ Raises:
     Count :: integer().
 observe_n(Registry, Name, LabelValues, Value, Count) when is_integer(Value), is_integer(Count) ->
     Key = key(Registry, Name, LabelValues),
-    try ets:lookup_element(?TABLE, Key, ?BOUNDS_POS) of
-        Buckets ->
+    case ets:lookup_element(?TABLE, Key, ?BOUNDS_POS, undefined) of
+        undefined ->
+            insert_metric(Registry, Name, LabelValues, Value, Count, fun observe_n/5);
```

Comment:
This seems like a fix, not adapting to a latest OTP. Where did this surface from?

Reply:
This one's taking advantage of a OTP26 feature, `ets:lookup_element/3` would raise if the element didn't exist. This was introduced here https://github.com/erlang/otp/commit/e528e26bf76267db4ff450d598c9a816d8d4a4ce and it's just a matter of being more ergonomic and also faster, no need to throw-catch.

Reply:
Cool stuff. I guess the title of the PR is a bit misleading then. 😄 You're not "upgrading to support", you're "adapting to recent enhancements" 👍 good stuff, in any case.